publications.md

```diff
@@ -10,5 +10,4 @@ The following is a list of publications coming from this project:
 D. Gopstein, J. Iannacone, Y. Yan, L. DeLong, Y. Zhuang, M. Yeh, J. Cappos<br />
 *Under Review*
 
-> Humans often mistake the meaning of source code, and so misjudge a program’s true behavior. To better under- stand code patterns that confuse programmers, we extracted a preliminary set from winners of the International Obfuscated C Code Contest (IOCCC). We show experimentally (using 73 participants) that most of these code patterns lead to a signifi- cantly increased rate of misunderstanding versus equivalent code without the pattern. We go on to empirically demonstrate (using 43 participants) that removing the design patterns that were validated as confusing from IOCCC winners has a significant effect on the level of comprehension. These studies provide not only a rigorous foundation to support software engineering best practices, but also resulted in several new findings. This study shows that extremely small patterns of code can have a very large impact on code comprehension. Surprisingly, the level of confusion an individual programmer has when reading these design patterns varies widely in ways that do not correlate with the programmer’s experience or education. We also found evidence that popular style guidelines are inclusive of some confusing patterns, while prohibiting others that do not have a statistically significant effect on confusion.
-
+> Humans often mistake the meaning of source code, and so misjudge a program’s true behavior. To better understand code patterns that confuse programmers, we extracted a preliminary set from winners of the International Obfuscated C Code Contest (IOCCC). We show experimentally (using 73 participants) that most of these code patterns lead to a significantly increased rate of misunderstanding versus equivalent code without the pattern. We go on to empirically demonstrate (using 43 participants) that removing the design patterns that were validated as confusing from IOCCC winners has a significant effect on the level of comprehension. These studies provide not only a rigorous foundation to support software engineering best practices, but also resulted in several new findings. This study shows that extremely small patterns of code can have a very large impact on code comprehension. Surprisingly, the level of confusion an individual programmer has when reading these design patterns varies widely in ways that do not correlate with the programmer’s experience or education. We also found evidence that popular style guidelines are inclusive of some confusing patterns, while prohibiting others that do not have a statistically significant effect on confusion.
```

Comment:
There were some odd breaks in a few words. That was all I changed here, I believe.